extensions/amp-fit-text/validator-amp-fit-text.protoascii

```diff
@@ -38,5 +38,6 @@ tags: {  # <amp-fit-text>
     supported_layouts: FLEX_ITEM
     supported_layouts: NODISPLAY
     supported_layouts: RESPONSIVE
+    supported_layouts: INTRINSIC
```

Comment:
/cc @Gregable @honeybadgerdontcare for validator and to ensure this does not break SSR given it is a new layout (also we need SSR support for this, see updated `applyStaticLayout` around the `layout == Layout.INTRINSIC` case)

Reply:
@jpettitt Can you file either an issue or a bug to implement instrinsic in SSR and assign to me?

Reply:
This shouldn't break SSR as it's designed to handle unknown layouts.